src/components/Builder/Builder.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, useRef, useEffect, memo } from 'react';
+import { Pathway, State } from 'pathways-model';
+
+import Header from 'components/Header';
+import Navigation from 'components/Navigation';
+import Sidebar from 'components/Sidebar';
+import Graph from 'components/Graph';
+import Loading from 'components/elements/Loading';
+
+import styles from './Builder.module.scss';
+
+interface BuilderProps {
+  pathway?: Pathway | null;
```

Comment:
I don't think should be optional anymore. What is the point in viewing the builder screen without a pathway? It isn't even accessible unless a pathway is provided anyway
```suggestion
  pathway: Pathway;
```